python/job_runner/reporting/event.py

```diff
@@ -87,7 +89,14 @@ def _init_handler(self, msg):
                     "datacontenttype": "application/json",
                 },
                 {
-                    "jobs": [job.job_data for job in msg.jobs],
+                    "jobs": {
+                        job.index: {
+                            **job.job_data,
+                            "stderr": os.path.abspath(job.job_data["stderr"]),
+                            "stdout": os.path.abspath(job.job_data["stdout"]),
+                        }
+                        for job in msg.jobs
```

Comment:
I find these multi line inline online dict expressions hard to read. Is it just me?

Reply:
Don't know, but let's try on another size and see how it fits.